Pipe.py

```diff
@@ -0,0 +1 @@
+print("pipe")
```

Comment:
Please provide the comments

Reply:
I agree with Tricharu please provide some commenrs